<!DOCTYPE html>
<html>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <link rel="stylesheet" type="text/css" href="../css/custom.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

    <style type="text/css">

      /* Sticky footer styles
      -------------------------------------------------- */
      html, body {
        max-width: 100%;
        overflow-x:hidden;
      }

      /* Wrapper for page content to push down footer */
      /* Negative indent footer by its height */
      /* Pad bottom by footer height */
      #wrap {
        min-height: 100%;
        min-width: 800px;
      }

      /* Set the fixed height of the footer here */
      #footer {
        height: 60px;
        text-align: center;
        font-size: 13px;
        padding-left:10px;
        padding-right:10px;
        margin-top: 15px;
        padding-top: 10px;
        border-top: 1px solid #DEDEDE;
        font-weight: 300;
      }

      #footer_text_container{
        margin-top: 10px;
        margin-bottom: 0px;
        min-width: 950px;
      }

      .clustergrammer_container{
        width: 1140px;
        height: 750px;
        margin-bottom: 10px;
        overflow: hidden;
      }

      .viz_title{
        width: 100%;
        font-size: 30px;
        font-weight: 300;
        /*margin-left: -10px;*/
        margin-top: 10px;
        margin-bottom: 5px;
        border-bottom: 1px solid #DEDEDE;
      }

      .viz_description{
        font-size: 16px;
        font-weight: 300;
        margin-top: 10px;
        margin-left: 5px;
        margin-right: 20px;
        text-align: justify;
        margin-bottom: 30px;
      }

    </style>

</head>
<body>

  <div id="wrap" class='container'>

    <div class='row viz_title' id='clustergram_title' style='white-space: nowrap; margin-left: 5px;'>
      <p id='example_title'>Omics Mining for the Illuminating the Druggable Genome Project</p>
    </div>

    <div class="container">
      <h2 style='font-weight: 300;'>Target Functional Predictions</h2>
      <p class='example_single_description'><b>Methods:</b> To predict functions for the under-studied kinases, GPCRs and ion channels we used a co-expression network created from RNA-seq data processed from GEO/SRA. An example is provided to show how biological processes from the gene ontology (GO) can be predicted. First, we processed all human and mouse RNA-seq samples currently (2/2017) available from GEO/SRA. Next, we calculated all pairwise gene correlations from the all the processed RNA-seq expression data. We then iterate over 5192 GO biological process terms where for each gene we calculated the average correlation with the genes in the gene set. The values are stored in a matrix C for all i genes and j biological processes. We then z-score normalized all the columns of C. Finally, for each gene in C we ranked terms based on their gene population z-score. Terms with high z-scores are predicted to be true annotations for the gene.</p>

      <!-- Figure A, B -->
      
      <div class="container" style="width: 800px;">
        <div class="row">

          <div  class='col-xs-6 example_col'>
            <img src="roc_go.png" align="middle" style='width: 350px;'>
          </div>

          <div  class='col-xs-6 example_col'>
            <img src="figure-from-alex-kinases.png" align="middle" style='width: 350px; margin-top: 12px;'>
          </div>

        </div>
      </div>

      <div class="container" style="width: 800px; margin-bottom:30px;">
        <div class="row">
          <div  class='col-xs-6 example_col' style="text-align: center;">
            A
          </div>
          <div  class='col-xs-6 example_col' style="text-align: center;">
            B
          </div>
        </div>
      </div>

      <div class="container" style="width: 800px;">
        <div class="row">

          <div  class='col-xs-6 example_col'>
            <img src="figure-from-alex-ion-channels.png" align="middle" style='width: 350px;'>
          </div>

          <div  class='col-xs-6 example_col'>
            <img src="figure-from-alex-gpcrs.png" align="middle" style='width: 350px;'>
          </div>

        </div>
      </div>

      <div class="container" style="width: 800px; margin-bottom:30px;">
        <div class="row">
          <div  class='col-xs-6 example_col' style="text-align: center;">
            C
          </div>
          <div  class='col-xs-6 example_col' style="text-align: center;">
            D
          </div>
        </div>
      </div>

      <p style="margin-bottom: 20px;">
      <b>Figure 1</b> a) To show the correctness of term ranking, area under the curve (AUC) from the ROC was calculated for all genes with existing annotations in GO. For GO biological processes, the average AUC for retrieving already annotated terms is 0.765. b-d) t-SNE plots of all human genes based on their co-expression similarity, where each gene is a point on the plot. Genes are colored based on the number of GO terms associated with each genes. Black dots are genes with no GO terms. Blue dots highlight the under-studied kinases, GPCRs and ion channels.
      </p>


      <p><b>Examples of GO annotations for the three under-studied kinase: PKMYT1, NEK4, PKN3; and the GPCR: ADGRD2</b></p>
    <table class="table table-hover">
    <tbody>
    <tr><td><a href="PKMYT1.html">PKMYT1</a><p>
    This gene encodes a member of the serine/threonine protein kinase family. The encoded protein is a membrane-associated kinase that was reported to negatively regulates the G2/M transition of the cell cycle by phosphorylating and inactivating cyclin-dependent kinase 1. The activity of the encoded protein is regulated by polo-like kinase 1. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. This kinase is listed as an under-studied kinase by IDG, and here we show the results of <a href="PKMYT1.html">predicting GO biological processes terms for PKMYT1.</a></p>
      </td></tr>
      
      <tr><td><a href="NEK4.html">NEK4</a>
    <p>NEK4 (NIMA Related Kinase 4) is a kinase known to be associated with Meckel Syndrome 1. It was reported that this kinase plays a role in sweet taste signaling. GO annotations related to this gene are few and include tyrosine kinase activity. This kinase is listed as an under-studied kinase by IDG, and here we show the results of <a href="NEK4.html">predicting GO biological processes terms for NEK4.</a></p>
    </td></tr>
    <tr><td><a href="PKN3.html">PKN3</a>
    <p>PKN3 (Protein Kinase N3) was reported to be a member of the Salmonella infection pathway (KEGG). GO annotations related to this gene include protein tyrosine kinase activity. This kinase is listed as an under-studied kinase by IDG, and here we show the results of <a href="PKN3.html">predicting GO biological processes terms for PKN3.</a></p>
      </td></tr>
      <tr><td><a href="ADGRD2.html">ADGRD2</a>
    <p>ADGRD2 (Adhesion G Protein-Coupled Receptor D2) is an under-studied G-protein coupled receptor. An important paralog of this gene is ADGRL1. This kinase is listed as an under-studied kinase by IDG, and here we show the results of <a href="ADGRD2.html">predicting GO biological processes terms for ADGRD2.</a></p>
    </td></tr>
    </tbody>
    </table>

    </div>


    <div id='footer'>
      <div class="row" >
        <div id='footer_text_container' class="col-xs-12 footer_section">
          <div class=" footer_text">Developed by the <a class='blue_links' target="_blank"  href="http://icahn.mssm.edu/research/labs/maayan-laboratory">Ma'ayan Lab</a> at the <a class='blue_links' target="_blank"   href="http://icahn.mssm.edu/">Icahn School of Medicine at Mount Sinai</a>  for the <a target="_blank" href="http://commonfund.nih.gov/idg/overview">KMC-IDG</a>  </div>
        </div>
      </div>
    </div>

  </div>

</body>
</html>
